tools/sgen/input/FluentUIStyle.yml

```diff
@@ -439,6 +440,12 @@ AP_MSFListTokens:
 
 MSFIconOnlyListTokens extends MSFListTokens:
   iconColor: $Colors.Foreground.neutral3
+  
+AP_MSFListHeaderFooterTokens:
+  backgroundColor: [ rest: $Colors.Background.neutral1, highlighted: $Colors.Background.neutral5 ]
+  textColor: [ default: $Colors.Foreground.neutral3, bolded: $Colors.Foreground.neutral1 ]
```

Comment:
I think default and bolded are good candidates for styles. Like the primary/ghost buttons for example.